ckan/lib/helpers.py

```diff
@@ -2344,6 +2344,35 @@ def mail_to(email_address, name):
     return html
 
 
+@core_helper
+def get_base_public_folder(app_conf):
```

Comment:
we can have it fail in update_config like it does when site_url isn't specified. (an exception other than RuntimeError would be better of course)